web-ui/src/components/view_feedback_responses/ViewFeedbackResponses.jsx

```diff
@@ -85,7 +86,21 @@ const ViewFeedbackResponses = () => {
           ? requests
           : [requests]
         : [];
-      return await getQuestionsAndAnswers(requests, cookie);
+      const res = await getQuestionsAndAnswers(requests, cookie);
+
+      if (res) {
+        console.log('Retrieved Questions and Answers:', res); // Debugging log
```

Comment:
as a rule, we try not to leave console.log calls in the codebase.

Reply:
Existing console.log calls in pertinent files have been removed.